docs/index.rst

```diff
@@ -3,77 +3,18 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to OmniPhotos's documentation!
-======================================
 
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
-
-
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
-
-Installation
-============
-Please see the REAME.md file for installation instructions. 
-
-Usage
-=====
-In order to synthesise new views from a dataset, the OmniPhotos pipeline needs to perform two steps.
-The first of these is to preprocess the dataset, the second is to use this preprocessed dataset to generate the views and render them on the screen or in a headset.
-We assume here that the software has been built/downloaded prior to these steps.
-
-The dataset I will be using for this demo will be that of KyotoShrines.
-In this dataset, there should be a config-viewer.yaml file located under the **Config/** directory, an **Input/** directory where the original dataset is located, a **Capture/** directory 
-where the structure-from-motion data is located, and possibly a **Cache/** directory, where the output from the preprocessing will be located.
-This last directory may not be located here yet, in which case it will be generated by the preprocessing step.
-
-
-Preprocessing
-#############
-In order to Preprocess the dataset using the **Preprocessing.exe** file, you will need to check that the **config-viewer.yaml** file is located in the correct directory.
-In this config file, there will be the name of the output cache file at the very top, which can be left as default.
-
-To run the exe file, simply run from the command line:
 
-   $ Preprocessing.exe /path-to-dataset/Config/config-viewer.yaml
+.. image :: representative_image.jpg
 
-where */path-to-dataset/* should be replaced with the relative or absolute path to the root directory of the dataset (in this case KyotoShrines).
-
-When you run this command, a dialogue may appear asking whether you would like to replace the existing preprocessed cache file.
-Answer "Yes" to this and let the program finish.
-
-
-Viewing
-#######
-We assume here that the dataset being used has been preprocessed.
-In order to run the exe file, simply run from the command line:
-
-   $ Viewer.exe /path-to-dataset/Config/config-viewer.yaml
-
-Wait until the program has finished initialising and the window has been launched.
-
-Viewer User Interface
-*********************
-
-The keyboard shortcuts for the viewer.
-
-.. include:: key-bindings-table.rst
-
-
-Docs
-====
+.. toctree::
+   :maxdepth: 3
+   :hidden:
+   :caption: Contents:
 
-.. doxygenclass:: ViewerApp
-   :members:
-   :private-members:
+|
 
-.. doxygenclass:: GLApplication
-   :members:
-   :private-members:
+########################################
+Welcome to the OmniPhotos documentation!
+########################################
+Not really.
```

Comment:
Do we still need this file?

Reply:
i believe so or sphinx will fail, meaning the RTD virtual machines will not complete the pipeline.

Reply:
Okay, fair enough. But I guess the file could be cleaned up a little.